### ISSUE TITLE
Enhancement: Enable the TableName invocation in the QueryMultiple operations

### ISSUE THREAD

**Issue by mikependon** [mikependon/RepoDb#1065]
### Describe the enhancement

Currently, only the generic type is supported by the [QueryMultiple](https://repodb.net/operation/querymultiple) operation. In order for this operation to be more dynamic and powerful when querying the data, the table name invocation must be supported.

```csharp
using (var connection = new SqlConnection(...))
{
	var orderId = 10045;
	var result = connection.QueryMultiple<Order, OrderItem>("Order", "OrderItem",
		o => o.Id == orderId, oi => oi.OrderId == orderId);
	var order = result.Item1.FirstOrDefault();
	var orderItems = result.Item2.ToList();
}
```

Where the `where` params can expect the following types.

- Anonymous Types
- IDictionary<string, object>
- QueryField
- IEnumerable<QueryField>
- QueryGroup

Where the anonymous type can expect the following values.

- .NET native type
- SqlDbDataParameter

**Comment by mikependon** [comment]
Referencing the #1070

**Comment by mikependon** [comment]
This will be available on the releases > RepoDb v1.12.10.

**Comment by mikependon** [comment]
The changes must also be implemented within the DbRepository object.
